resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -464,19 +464,22 @@ ecs.execute_command.label=Command:
 ecs.execute_command.production_warning.checkbox_label=I confirm {0} is not a production service and wish to continue
 ecs.execute_command_disable=Disable Command Execution
 ecs.execute_command_disable_failed=Execute Command could not be disabled for {0}
+ecs.execute_command_disable_in_progress=Cannot Execute Action since Execute Command is being disabled for {0}
 ecs.execute_command_disable_progress_indicator_message=Disabling Command Execution for {0}
 ecs.execute_command_disable_success=Execute Command for {0} has been disabled.
 ecs.execute_command_disable_warning=Disabling command execution will change the state of resources in your AWS account, including but not limited to stopping and restarting the service.\n Do you wish to continue?
 ecs.execute_command_disable_warning_title=Disable Execute Command Warning
 ecs.execute_command_enable=Enable Command Execution
 ecs.execute_command_enable_failed=Execute Command could not be enabled for {0}
+ecs.execute_command_enable_in_progress=Cannot Execute Action since Execute Command is being enabled for {0}
 ecs.execute_command_enable_progress_indicator_message=Enabling Command Execution for {0}
 ecs.execute_command_enable_success=Execute Command for {0} has been enabled.
 ecs.execute_command_enable_warning=Enabling command execution will change the state of resources in your AWS account, including but not limited to stopping and restarting the service.\nAltering the state of resources while the Execute Command is enabled can lead to unpredictable results.\n Do you wish to continue?
 ecs.execute_command_enable_warning_title=Enable Execute Command Warning
 ecs.execute_command_no_command=Command to execute must be entered
 ecs.execute_command_parameters=ecs execute-command --cluster {0} --task {1} --command {2} --interactive --container {3}
 ecs.execute_command_run=Run Command...
+ecs.execute_command_run_command_default_text=Enter Command...
```

Comment:
command should be lower cased, it is not a title

Reply:
done